MovieChecker/MovieChecker/Info.plist

```diff
@@ -22,8 +22,6 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-	<key>UILaunchStoryboardName</key>
-	<string>LaunchScreen</string>
```

Comment:
If you're not using the LaunchScreen xib, you can probably delete it from the project.